LanguageExt.Core/Traits/Monads/MonadIO/MonadIO.Trait.cs

```diff
@@ -44,17 +44,6 @@ static K<M, A> Maybe.MonadIO<M>.LiftIOMaybe<A>(IO<A> ma) =>
 
     static K<M, A> Maybe.MonadIO<M>.LiftIOMaybe<A>(K<IO, A> ma) => 
         M.LiftIO(ma);
-
-    public static virtual K<M, C> SelectMany<A, B, C>(K<M, A> ma, Func<A, IO<B>> bind, Func<A, B, C> project) =>
```

Comment:
This (and the below) where not used anywhere, perhaps they were intended to do what `resolve` is now doing?